docs/source/contrib_docs/doc-bugs.rst

```diff
@@ -0,0 +1,2 @@
+Correcting bugs
```

Comment:
What are these files for? Are they placeholders for something that's going to be there, or leftovers from something removed so that links don't break?